### PR TITLE
Add site creation and naming info (SOFTWARE-3894)

### PR DESCRIPTION
**Pull request by matyasselmeci**
The ticket was for adding site naming recommendations, but I couldn't
find a way to make it make sense without also having instructions for
creating a site.

### CODE REVIEW

File: docs/common/registration.md [hunk 1]
```diff
@@ -86,14 +86,23 @@ host to avoid any duplicate registrations:
 
 ### New resources ###
 
+Before registering a new resource, make sure that its FQDN is not [already registered](#searching for resources).
+
 To register a new resource, follow the instructions below:
 
-1. If you haven't already, verify that the FQDN of your resource is not [already registered](#searching-for-resources)
+1. Find the facility, site, and resource group for your resource.
```
Comment (edquist) [inline docs/common/registration.md]: Note that this removes the instruction to verify the FQDN is not already registered.  Is this step no longer relevant?  (My impression is it's still relevant.)

File: docs/common/registration.md [hunk 2]
```diff
@@ -176,6 +187,39 @@ Resources:
 If the `Active` attribute does not already exist within the resource definition, add it.
 If your resource becomes available again, set `Active: true`.
 
+
+### Registering a site
```
Comment (brianhlin) [inline docs/common/registration.md]:
```suggestion
### New site
```

Let's move this section above the `Searching for resources` section.

File: docs/common/registration.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ OSG resources are stored under a hierarchy of facilities, sites, and resource gr
 | Level          | Definition                                                                                                                                                                    |
 |----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Facility       | The institution or company where your resource is located, e.g. `University of Wisconsin`                                                                                     |
-| Site           | Smaller than a facility; typically represents an academic department, research group, or a computing center, e.g. `CHTC` for the Center for High Throughput Computing.        |
+| Site           | Smaller than a facility; typically represents a computing center or an academic department, e.g. `CHTC` for the Center for High Throughput Computing.                         |
```
Comment (brianhlin) [inline docs/common/registration.md]:
```suggestion
| Site           | Smaller than a facility; typically represents a computing center or an academic department, e.g. `UW-CHTC` for the Center for High Throughput Computing.                         |
```
Updated for the new naming convention

**Comment by matyasselmeci** [comment]
I moved the section; how does it look now?

**Pull request merged**